.drone.yml

```diff
@@ -47,3 +47,97 @@ steps:
     branch:
     event:
     - pull-request
+---
+kind: pipeline
+name: build-arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: build
+  pull: always
+  image: golang:1.13
```

Comment:
Any reason not to use 1.14?

Reply:
just copy paste from old version of drone.yaml